src/library/Uno.Material/Styles/Controls/v2/ToggleSwitch.xaml

```diff
@@ -36,31 +13,142 @@
+			<StaticResource x:Key="ToggleSwitchKnobOffShadowFillFocused" ResourceKey="OnSurfaceVariantFocusedBrush" />
+			<StaticResource x:Key="ToggleSwitchKnobOffShadowFillPressed" ResourceKey="OnSurfaceVariantPressedBrush" />
+			<StaticResource x:Key="ToggleSwitchOnSwitchKnobBoundsFill" ResourceKey="PrimaryBrush" />
+			<StaticResource x:Key="ToggleSwitchKnobBoundsFillNormal" ResourceKey="PrimaryBrush" />
+			<StaticResource x:Key="ToggleSwitchKnobBoundsFillPointerOver" ResourceKey="PrimaryBrush" />
+			<StaticResource x:Key="ToggleSwitchKnobBoundsFillFocused" ResourceKey="PrimaryBrush" />
+			<StaticResource x:Key="ToggleSwitchKnobBoundsFillPressed" ResourceKey="PrimaryBrush" />
+			<StaticResource x:Key="ToggleSwitchKnobBoundsFillDisabled" ResourceKey="OnSurfaceDisabledBrush" />
+			<StaticResource x:Key="ToggleSwitchOffIconPresenterForeground" ResourceKey="SurfaceVariantBrush" />
+			<StaticResource x:Key="ToggleSwitchOnIconPresenterForeground" ResourceKey="OnPrimaryContainerBrush" />
+			<StaticResource x:Key="ToggleSwitchOnIconPresenterForegroundDisabled" ResourceKey="OnSurfaceLowBrush" />
+			<StaticResource x:Key="ToggleSwitchOffIconPresenterForegroundDisabled" ResourceKey="SurfaceVariantBrush" />
+			<StaticResource x:Key="ToggleSwitchThumb" ResourceKey="SystemControlTransparentBrush" />
+
+			<x:Double x:Key="SwitchKnobWidth">52</x:Double>
+			<x:Double x:Key="SwitchKnobIncludingOffShadowWidth">56</x:Double>
+			<x:Double x:Key="SwitchKnobHeight">32</x:Double>
+			<x:Double x:Key="SwitchKnobRadius">16</x:Double>
+			<x:Double x:Key="SwitchKnobStrokeThickness">2</x:Double>
+			<Thickness x:Key="SwitchKnobOnMargin">26,0,0,0</Thickness>
+			<Thickness x:Key="SwitchKnobOffMargin">0,0,22,0</Thickness>
+
+			<x:Double x:Key="SmallThumbSize">16</x:Double>
+			<x:Double x:Key="MediumThumbSize">24</x:Double>
+			<x:Double x:Key="LargeThumbSize">28</x:Double>
+
+			<CornerRadius x:Key="LargeThumbCornerRadius">14</CornerRadius>
+
+			<x:Double x:Key="KnobIconSize">16</x:Double>
+			<Thickness x:Key="KnobIconPadding">2</Thickness>
+			<Thickness x:Key="KnobOnMargin">2,0,0,0</Thickness>
+
+			<x:Double x:Key="SwitchKnobShadowSize">40</x:Double>
+			<Thickness x:Key="SwitchKnobOnShadowMargin">20,0,0,0</Thickness>
+			<Thickness x:Key="SwitchKnobOffShadowMargin">0,0,20,0</Thickness>
+		</ResourceDictionary>
+		<ResourceDictionary x:Key="Light">
+			<!-- MaterialToggleSwitchStyle -->
+			<StaticResource x:Key="ToggleSwitchOffOuterBorderFill" ResourceKey="SurfaceVariantBrush" />
+			<StaticResource x:Key="ToggleSwitchOffOuterBorderStroke" ResourceKey="OutlineBrush" />
+			<StaticResource x:Key="ToggleSwitchOuterBorderStrokeNormal" ResourceKey="OutlineBrush" />
+			<StaticResource x:Key="ToggleSwitchOuterBorderFillNormal" ResourceKey="SurfaceVariantBrush" />
+			<StaticResource x:Key="ToggleSwitchKnobOffFill" ResourceKey="OnSurfaceVariantBrush" />
+			<StaticResource x:Key="ToggleSwitchKnobOffFillNormal" ResourceKey="OnSurfaceVariantBrush" />
```

Comment:
The "Normal" VisualState is a special case where we don't need a separate resource with "Normal" attached at at the. In the Normal VisualState we should just be using "ToggleSwitchKnobOffFill" so there's no need to have two resources here with the same value, they can be merged into one resource without the "Normal" at the end end of the name